docs/getting-started/index.md

```diff
@@ -78,12 +79,21 @@ yarn dev
 It might take a little while, but as soon as the message
 `[0] webpack compiled successfully` appears, you can open a browser and directly
 navigate to your freshly installed Backstage portal at `http://localhost:3000`.
-You can start exploring the demo immediately.
+You can start exploring the demo immediately. Please note that the in-memory
+database will be cleared when you restart the app, so you'll most likely want to
+carry on with the database steps.
 
 <p align='center'>
   <img src='../assets/getting-started/portal.png' alt='Screenshot of the Backstage portal.'>
 </p>
 
+The most common next steps are to configure Backstage, add a plugin and moving
+to a persistent database:
+
+- [Setting up Authentication](https://backstage.io/docs/auth/)
+- [Switching from SQLite to PostgresQL](https://backstage.io/docs/tutorials/switching-sqlite-postgres)
+- [Adding a plugin](https://backstage.io/docs/getting-started/configure-app-with-plugins)
```

Comment:
based on the sentence above maybe switch the order for the database and the plugin steps, just for the numeric alignment (1, 2, 3)....